SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialDispatcher.cpp

```diff
@@ -197,3 +169,121 @@ Worker_ComponentId USpatialDispatcher::GetComponentId(Worker_Op* Op) const
 		return SpatialConstants::INVALID_COMPONENT_ID;
 	}
 }
+
+USpatialDispatcher::FCallbackId USpatialDispatcher::AddOpCallback(Worker_ComponentId ComponentId, const TFunction<void(const Worker_AddComponentOp&)>& Callback)
```

Comment:
Why not follow the Worker SDK dispatcher naming for these methods instead of overloading - OnAddComponent, OnAuthorityChange, OnCommandRequest, etc.? This is also consistent with the connection events - OnConnected for example.